Dependencies/JOSESwift.xcframework/Info.plist

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you automate the creation of xcframework of JoseSwift for future upgrades if required?

Reply:
Thanks, In the meantime can you please document the steps if we need to upgrade the JoseSwift manually till we have automation in place?. Not a blocker for PR.